### PR TITLE
React Select. Support `CreatableSelect` component

### PR DESCRIPTION
**Pull request by aerialist7**

### CODE REVIEW

File: kotlin-react-select/src/jsMain/kotlin/react/select/GetOptionLabel2.kt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package react.select
+
+import react.ReactNode
+
+typealias GetNewOptionData<Option> = (
```
Comment (turansky) [inline kotlin-react-select/src/jsMain/kotlin/react/select/GetOptionLabel2.kt:5]: Invalid file name

File: kotlin-react-select/src/jsMain/kotlin/react/select/CreatableSelect.ext.kt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package react.select
+
+import js.objects.jso
+import react.ChildrenBuilder
+import react.ReactDsl
+
+fun <Option : Any, Group : GroupBase<Option>> ChildrenBuilder.CreatableSelect(
```
Comment (turansky) [inline kotlin-react-select/src/jsMain/kotlin/react/select/CreatableSelect.ext.kt:7]: Adapter is redundant with new builder

**Review by turansky — CHANGES_REQUESTED** [review]

File: kotlin-react-select/src/jsMain/kotlin/react/select/GetOptionLabel2.kt (hunk 1 above)
Comment (aerialist7) [reply kotlin-react-select/src/jsMain/kotlin/react/select/GetOptionLabel2.kt]: Fixed

File: kotlin-react-select/src/jsMain/kotlin/react/select/CreatableSelect.ext.kt (hunk 2 above)
Comment (aerialist7) [reply kotlin-react-select/src/jsMain/kotlin/react/select/CreatableSelect.ext.kt:7]: I tried to pass generic type directly to `external val CreatableSelect` but it doesn't work.
How it should looks like?

Comment (turansky) [reply kotlin-react-select/src/jsMain/kotlin/react/select/CreatableSelect.ext.kt:7]: You can find example in `mui-kotlin` playground

Comment (aerialist7) [reply kotlin-react-select/src/jsMain/kotlin/react/select/CreatableSelect.ext.kt]: Done, thanks!

**Review by turansky — APPROVED** [review]

**Comment by aerialist7** [comment]
@Leonya, could you please make a release?

**Pull request merged**